a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -88,17 +87,19 @@ def _set(self, set_method, c, context, listener):
                 # If the binding is being deleted and the port isn't https
                 # remove the port and re-create it.
                 if binding.status == constants.STATUS_DELETING:
+                    import pdb
+                    pdb.set_trace()
+
                     if (protocol != c.client.slb.virtual_server.vport.HTTPS):
-                        self._delete_listener(c, context, listener)
+                        self._delete_listener(c, context, listener, listener.protocol)
```

Comment:
This is passing an openstack protocol...

`_delete_listener` takes an acos protocol.